CodeStarterCamp_Week5/auditionManager.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  auditionManager.swift
+//  CodeStarterCamp_Week5
+//
+//  Created by MARY on 2023/03/20.
+//
+
+import Foundation
+
+extension Talent {
+    var hasOutstandingTalent: Bool {
+        get {
+            self.singing == Level.A || self.dancing == Level.A || self.acting == Level.A
+        }
+    }
+}
+
+extension BadPersonality {
+    var isBadPerson: Bool {
+        get {
+            self.frequancyOfCursing == Level.A
+        }
+    }
+}
```

Comment:
프로토콜 기본 구현을 사용 하셨군요! 👍👍

다만, 이 프로토콜을 채택하는 타입은 노래, 춤, 연기, 비주얼의 재능과 함께 `합격할만한 재능인지 판별하는` 능력도 생기겠네요?
그리고, 인성 나쁜사람은 `자신의 인성이 얼마나 나쁜지 판별하는` 능력도 생길 것 같아요.

합격자를 선별하는 객체는 `Talent`프로토콜을 채택한 객체일까요? 🤔
나쁜 인성을 가진 사람을 걸러내는 객체는 `BadPersonality`프로토콜을 채택한 객체일까요?
이대로라면 지원자가 스스로 합격여부를 체크하는 모습일 것 같아요. 맞는 방향성의 설계인지 살짝 의문이 듭니다!

틀렸다는 뜻은 절대 아닙니다! 가독성은 훨씬 좋아진건 맞으니까요! 따라서 꼭 수정하실 필요도 없습니다.
저였다면 이 체크하는 로직을 `AuditionManager`의 메서드로 넣을거 같긴 해요. 😅😅
한번 어느 방향성이 맞을지 고민해보시기 바랍니다!

추가적으로, 연산 프로퍼티에서 get만 구현하신다면, 이를 생략하시는것도 가능합니다. ㅎㅎ
```suggestion
extension Talent {
    var hasOutstandingTalent: Bool {
        self.singing == Level.A || self.dancing == Level.A || self.acting == Level.A
    }
}

extension BadPersonality {
    var isBadPerson: Bool {
        self.frequancyOfCursing == Level.A
    }
}
```